x/harvest/keeper/borrow.go

```diff
@@ -11,6 +11,17 @@ import (
 
 // Borrow funds
 func (k Keeper) Borrow(ctx sdk.Context, borrower sdk.AccAddress, coins sdk.Coins) error {
+	// Set any new denoms' global borrow index to 1.0
+	for _, coin := range coins {
+		_, foundBorrowIndex := k.GetBorrowIndex(ctx, coin.Denom)
+		if !foundBorrowIndex {
+			k.SetBorrowIndex(ctx, coin.Denom, sdk.OneDec())
+		}
+	}
+
+	// Sync user's borrow balance (only for coins user is requesting to borrow)
+	k.SyncBorrowInterest(ctx, borrower, coins)
```

Comment:
This looks good. Add issue to sync interest for all borrows rather than just `coins` for follow-on PR.

Reply:
Added an Asana card